### PR TITLE
Add jest and storyshots

### PR DESCRIPTION
**Pull request by clemensp**

### CODE REVIEW

File: components/src/Box/Box.story.js [hunk 1]
```diff
@@ -1,16 +1,14 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
-import { withInfo } from "@storybook/addon-info";
+// import { withInfo } from "@storybook/addon-info";
```
Comment (matthewlyle) [inline components/src/Box/Box.story.js]: this could be removed instead of commented out.

**Comment by matthewlyle** [comment]
What is the point of the SpecHelper.js? Do we plan on putting things in it?

**Review by matthewlyle — APPROVED** [review]

File: components/src/Box/Box.story.js (hunk 1 above)
Comment (clemensp) [reply components/src/Box/Box.story.js]: Removed.

**Comment by clemensp** [comment]
> What is the point of the SpecHelper.js? Do we plan on putting things in it?

It's the place to put setup/configuration that is shared across specs.
There used to be a line in there, but I removed it because it wasn't needed.

I think the file is still nice to have to provide that central spot for setups, so I'll leave it.

**Pull request merged**